### ISSUE TITLE
Waterlevel 0

### ISSUE THREAD

**Issue by julesair** [luethi/Discharge_Optical_Measurement#33]
Hey photrack Team,

John from CETRAD reported that they can not process 0-waterlevel data with the App, thus also not upload it to the database. At the moment most of their rivers and furrows are dry and it is important to have this record in the database as well.
Processing a video with 0 waterlevel should just output 0-discharge. When one tries to do that at the moment, the App crashes.

**Comment by salpeha** [comment]
Hi Jules,

I tried with my phone and the app does not crash, but of course, for 0 water level it cannot calculate. Which phone are you using?
In any case for reporting a 0 water level you can use the functionality "collect discharge data"

**Comment by julesair** [comment]
Hey Slavador,

The App is actually not crashing, sorry that was a misunderstanding. I could not try it out myself. But as you said, it does not process the video, what makes sense. 
The collect function is the easiest way to go, that is true. On the other hand one does not have a proof image in that case. In my opinion the following procedure would be more suitable: The user records a video, he/she sets waterlevel 0, the app recognizes the 0-waterlevel,skips processing and just uploads discharge=0 + proof image. What do you think?

**Comment by salpeha** [comment]
Hi Jules, I agree. Actually already yesterday asked Ranko about the possibility to add that functionality.

**Comment by timoherberz** [comment]
Hi everyone,
we are having the same problem in Rufiji Basin, Tanzania.
One of our crowd-senders measured an intake, that was closed at that moment which means that the water was standing still. When trying to process the video, the processing fails.
I uploaded two videos for the site "Mlowo @ Inyala B intake" in the organisation "Rufiji Basin". As Jules said, this is an issue, as zero flow data is also important. Especially because for us it looks like the crowd-sender is not measuring at all, as there is no data uploaded.
Our crowd-senders are very unexperienced with smartphones and I would prefer not to teach them a new procedure. As the water level in this case is not 0, but the flow is zero, I don't feel, that setting the waterlevel to 0 is the best solution.
Is it possible that the app recognizes that there is no movement and gives as output 0 flow? That would be a good solution in my point of view.
Warm regards, Timo
